apps/payments/next/app/layout.tsx

```diff
@@ -20,19 +18,6 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body>
-        <header
-          className="bg-white fixed flex justify-between items-center shadow h-16 left-0 top-0 mx-auto my-0 px-4 py-0 w-full z-40 tablet:h-20"
-          role="banner"
-          data-testid="header"
-        >
-          <Image
-            src={mozillaLogo}
-            alt="Mozilla logo"
-            className="object-contain"
-            data-testid="branding"
-            width={120}
-          />
-        </header>
         <main className="mt-16 min-h-[calc(100vh_-_4rem)]">{children}</main>
```

Comment:
I'm seeing extra space below the header - I think `min-h-[calc(100vh_-_4rem)] tablet:mt-4` would fix this.

Branch
<img width="1078" alt="Screenshot 2025-01-24 at 2 28 53 PM" src="https://github.com/user-attachments/assets/5f2499da-6366-4294-8596-226cce5c10dc" />

Stage
<img width="1084" alt="Screenshot 2025-01-24 at 2 29 16 PM" src="https://github.com/user-attachments/assets/cbc7d0cc-3dff-414e-bda7-1326981d0665" />

Reply:
There was an unnecessary ` className="mt-16 min-h-[calc(100vh_-_4rem)]"` on the [locale]/layout.tsx `<main>` tag which was causing this gap, removing it fixed the issue!

Reply:
![Screenshot 2025-01-24 at 4 50 03 PM](https://github.com/user-attachments/assets/d63d44eb-1e67-43e5-8a84-d98f169e444f)